app/wama.web/Views/AdminConsole/UserTool/ViewCertifications.cshtml

```diff
@@ -9,51 +9,59 @@
 @{
     var controllerName = nameof(UserToolController).StripController();
     string MemberID = ViewBag.MemberId;
-    @Html.Partial("DisplayError")
-    <div class="col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
-        <div style="font-size:x-large">
-            <a href="@Url.Action(controller: controllerName,
-                            action: nameof(UserToolController.ViewAccount),
-                            values: new { MemberId = MemberID })">
-                Info
-            </a>
-            | <a href="@Url.Action(controller: controllerName,
-                                action: nameof(UserToolController.ViewCertifications),
-                                values: new { MemberId = MemberID })">
-                <u>Certification</u>
-            </a>
-            <br />
-            <br />
-        </div>
 
-        <div>
-            <table class="table table-bordered" id="@Constants.DATA_TABLE">
-                <thead>
-                    <tr>
-                        <th>@Html.LabelFor(x => x.ElementAt(0).Type)</th>
-                        <th>@Html.LabelFor(x => x.ElementAt(0).CertifiedOn)</th>
-                        <th>@Html.LabelFor(x => x.ElementAt(0).ExpiresOn)</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    @foreach (var item in Model)
-                    {
+    if (object.Equals((ViewBag.AccountType).AccountType, UserAccountType.Patron))
```

Comment:
If you choose my 2nd suggested code, then you do something like this.

`if ((bool)(ViewBag.IsPatron??false))`